samples/subsys/ipc/openamp/remote_echo/boards/udoo_neo_full.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_IPM_IMX=y
+CONFIG_IPM_IMX_MAX_DATA_SIZE_4=y
```

Comment:
Could you please rename the file to udoo_neo_full_m4.conf?

Reply:
addressed